compiler/fir/checkers/gen/org/jetbrains/kotlin/fir/analysis/diagnostics/FirErrors.kt

```diff
@@ -146,6 +147,7 @@ object FirErrors {
     val INAPPLICABLE_CANDIDATE by error1<FirSourceElement, PsiElement, AbstractFirBasedSymbol<*>>(SourceElementPositioningStrategies.REFERENCE_BY_QUALIFIED)
     val INAPPLICABLE_LATEINIT_MODIFIER by error1<FirSourceElement, KtModifierListOwner, String>(SourceElementPositioningStrategies.LATEINIT_MODIFIER)
     val VARARG_OUTSIDE_PARENTHESES by error0<FirSourceElement, KtExpression>()
+    val NAMED_ARGUMENTS_NOT_ALLOWED by error1<FirSourceElement, KtSimpleNameExpression, BadNamedArgumentsTarget>()
```

Comment:
This is `PsiElement` in FE1.0 instead of `KtSimpleNameEXpression`

Reply:
Oops!